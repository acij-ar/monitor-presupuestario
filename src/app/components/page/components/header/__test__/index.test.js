const React = require('react');
const { shallow } = require('enzyme');

const Header = require('..');

describe('Header component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
