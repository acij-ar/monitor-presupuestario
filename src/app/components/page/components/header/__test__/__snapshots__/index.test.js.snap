// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should match snapshot 1`] = `
<div
  id="header"
>
  <div
    id="header-content"
  >
    <a
      href="/"
      id="header-logo"
    />
    <input
      id="header-menu-toggle"
      type="checkbox"
    />
    <label
      htmlFor="header-menu-toggle"
      id="header-menu"
    />
    <ul
      id="header-menu-links"
    >
      <li
        id="home"
      >
        <a
          href="/"
        >
          INICIO
        </a>
      </li>
      <li
        id="monitor"
      >
        <a
          href="/monitor"
        >
          MONITOR
        </a>
      </li>
      <li
        id="about"
      >
        <a
          href="/acerca-de"
        >
          ACERCA DE
        </a>
      </li>
      <li
        id="twitter"
      >
        <a
          href="https://twitter.com/ACIJargentina"
          rel="noopener noreferrer"
          target="_blank"
        >
          <img
            alt="twitter"
            src="/static/twitter.svg"
          />
        </a>
      </li>
      <li
        id="facebook"
      >
        <a
          href="https://www.facebook.com/ACIJ.ORG"
          rel="noopener noreferrer"
          target="_blank"
        >
          <img
            alt="facebook"
            src="/static/facebook.svg"
          />
        </a>
      </li>
      <li
        id="github"
      >
        <a
          href="https://github.com/acij-ar/monitor-presupuestario"
          rel="noopener noreferrer"
          target="_blank"
        >
          <img
            alt="github"
            src="/static/github.svg"
          />
        </a>
      </li>
    </ul>
  </div>
</div>
`;
